lib/kamal/commands/builder/base.rb

```diff
@@ -33,7 +34,7 @@ def info
   end
 
   def inspect_builder
-    docker :buildx, :inspect, builder_name unless docker_driver?
+    docker :buildx, :inspect, builder_name unless docker_driver? || pack?
```

Comment:
Maybe we could extract a `buildx?` method here?

```
def buildx?
  !docker_driver? && !pack?
end
```

Reply:
@djmb We could also run `pack builder inspect` which returns a bunch of information about the default builder. It's a lot of information but might be useful to help triage if you're not sure what builder you're using. The Pack CLI lets you set your default builder so I have mine set to `heroku/builder:24` via `pack config default-builder heroku/builder:24`

Reply:
Does this mean I can now pass my builder name to kamal?

Buildx cloud_builder

Reply:
@alexohre No, I don't think there's a PR open for that, just the discussion here https://github.com/basecamp/kamal/discussions/914#discussioncomment-10455627